frontends/verilog/verilog_parser.y

```diff
@@ -1532,27 +1532,27 @@ cell_port_list_rules:
 	cell_port | cell_port_list_rules ',' cell_port;
 
 cell_port:
-	/* empty */ {
+	attr {
```

Comment:
You are leaking the `std::map` pointer returned by attr. Please add `free_attr($1);` to all the cases where add an instance of `attr` that you just want ignored.

Reply:
You're right. I pushed a fix.